### PR TITLE
native: fix contact activity showing up in feed

### PR DESCRIPTION
**Pull request by latter-bolden**
OTT

Fixes TLON-3345 by always permitting contact update events to be included while bundling (irrespective of notify status).

Also updates the logic to include contact events in separate "activity bundles". Since we don't roll them up on the client, we need to treat them as distinct sources to avoid misrepresenting the timeline (similar to how we treat mentions).

### CODE REVIEW

File: desk/app/activity.hoon [hunk 1]
```diff
@@ -590,7 +590,7 @@
   ::  we only care about posts/replies events that are notified, and we
   ::  don't want to include events from sources whose latest event is
   ::  after the start so we always get "new" sources when paging
-  ?.  ?&  notified.event
+  ?.  ?&  ?|(notified.event ?=(%contact -<.event))
```
Comment (mikolajpp) [inline desk/app/activity.hoon:593]: Seems like this circumvents the intended logic, but @arthyn approved this for now.

File: desk/app/activity.hoon [hunk 2]
```diff
@@ -613,6 +613,14 @@
   ?^  mention
     :-  sources.acc
     [(sub limit.acc 1) (snoc happenings.acc u.mention) collapsed.acc]
+  =/  contact-bundle=(unit activity-bundle:a)
+    ?.  ?=(%all type)  ~
+    =/  is-contact-event  ?=(%contact -<.event)
+    ?.  is-contact-event  ~
```
Comment (mikolajpp) [inline desk/app/activity.hoon:619]: This should be written as one line as the variable is not used anywhere
```suggestion
    ?.  ?=(%contact -<.event)  ~
```

**Review by mikolajpp — CHANGES_REQUESTED** [review]
One nit, otherwise is good to go.

File: desk/app/activity.hoon (hunk 1 above)
Comment (arthyn) [reply desk/app/activity.hoon:593]: yeah I think this is an ok compromise for now. what we really want are volume settings related to "where" an activity item should show up like unreads, feed, push notifications, etc. we had talked about it when first building activity, but it had already gone long so we decided to go with what we have.

**Pull request merged**